export function foo(): number {
  return 100
}

try {
  console.log(foo())
} catch (err) {}
